coursier.bzl

```diff
@@ -588,6 +590,16 @@ def _deduplicate_artifacts(dep_tree):
     dep_tree.update({"dependencies": deduped_artifacts.values() + null_artifacts})
     return dep_tree
 
+def _check_artifacts_are_unique(artifacts):
```

Comment:
nit improvements, feel free to change in a follow-up.

* Only print warning messages for each set of duplicated artifacts once. I don't think there's a need for 3 identical messages for a set of 3 duplicated artifacts.
* Show the full set of duplicated artifacts including versions. e.g. if com.foo.bar has v1, v2 and v3 in the list, print info about v1, v2 and v3 as well.